source/path.js

```diff
@@ -11,7 +13,7 @@ import paths from './paths';
  * @typedefn Idx = String | Int
  * @sig [Idx] -> {a} -> a | Undefined
  * @param {Array} path The path to use.
- * @param {Object} obj The object to retrieve the nested property from.
+ * @param {Object} obj The object or array to retrieve the nested property from.
  * @return {*} The data at `path`.
  * @see R.prop, R.nth
```

Comment:
We should also mention `assocPath`/`dissocPath` here.

Reply:
Done.